### DISCUSSION TITLE
레벨2 5주차 회고

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#96] [General]
# 레벨2 5주차 회고

🌈 이번 주의 생각 💭 토리와 미션하기 어쩌다보니 토리와 페어가 되었다~~ 브라운의 가호에 감사를 😎 이번 미션이 너무 어려웠는데 토리랑 함께여서 어려운 설계도 금방금방 변경할 수 있었다. 사실 이번 주에 약속도 많고 특히 미션하는 화요일, 수요일에 할 일이 많았다. 수요일에는 저녁에 남아서 하지도 못해서 제출 못할까봐 좀 두렵긴 했다 ㅋㅋ 그럼에도 다 …

https://vero.wiki/level2-fifth-retrospect/

**Comment by hong-sile** [comment]
와 그 술자리 쉽지 않아
